content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,328 @@
+new changes in the `v1beta1` authorization policy.
+
+An `AuthorizationPolicy` includes a `selector` and a list of `rule`. The `selector`
+is used to decide where to apply the policy, the `rule` is used to decide **who**
+could do **what** under which **conditions**.
+
+The `selector` replaces the functionality provided by `ClusterRbacConfig`
+and the `services` field in `ServiceRole`. The `rule` includes `from`, `to` and
+`when` which replaces the other fields in `ServiceRole` and `ServiceRoleBinding`.
+
+### Example
+
+The following authorization policy applies to workloads with label `app: httpbin`
+and `version: v1` in namespace `foo`:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: httpbin
+ namespace: foo
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.headers[version]
+     values: ["v1", "v2"]
+{{< /text >}}
+
+The policy allows principal `cluster.local/ns/default/sa/sleep` to access the workload
+using `GET` method when the request includes a `version` header of value `v1` or `v2`.
+Any requests not matched with the policy will be denied by default.
+
+That's all you need to enable access control for a workload, compared to the `v1alpha1`
+RBAC policy, you would need to configure three resources to achieve the same result:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ClusterRbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    services: ["httpbin.foo.svc.cluster.local"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  rules:
+  - services: ["httpbin.foo.svc.cluster.local"]
+    methods: ["GET"]
+    constraints:
+    - key: request.headers[version]
+      values: ["v1", "v2"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: httpbin
+  namespace: foo
+spec:
+  subjects:
+  - user: "cluster.local/ns/default/sa/sleep"
+  roleRef:
+    kind: ServiceRole
+    name: "httpbin"
+{{< /text >}}
+
+### Workload selector
+
+One major change in the `v1beta1` authorization policy is it now uses workload selector
+to specify where to apply the policy. This is the same workload selector used in
+the `Gateway`, `Sidecar` and `EnvoyFilter` configurations.
+
+The workload selector makes it clear that the policy is applied and enforced on workloads
+instead of services. If a policy applies to a workload that is used by multiple different
+services, the same policy will affect the traffic to all the different services.
+
+You can simply leave the `selector` empty to apply the policy to all workloads
+in a namespace. The following policy applies to all workloads in the namespace `bar`:
```

Comment:
it is good to highlight which rules (or all the rules) the workload selector applies to.  I don't think it would apply to from or when.

Reply:
I updated this in the beginning of the blog, should be clear that all rules will have effect in this case.